### DISCUSSION TITLE
GH yaml editor does not recognize custom shell

### DISCUSSION THREAD

**Discussion by andreasplesch** [community/community#25148] [Actions]
<p><img src="https://user-images.githubusercontent.com/6171115/181089154-cd195234-9421-4122-8842-54bbe45c1e61.png" alt="image.png" data-base62-sha1="aymsYCoPkIISkw3se75zRTlHmE5" width="384" height="156">The Github online editor complains about custom shell parameters (they work fine, of course).</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/andreasplesch">@andreasplesch</a> , </p>
<p>The shell keyword in a workflow yml file on the github online yaml editor could only recognize suggested shell names. Custom shells are not included in the suggestions.  </p>
<p><img src="https://user-images.githubusercontent.com/24806478/181089156-6e027142-7d8f-4a9a-a502-0ebeecaf60c3.png" alt="bash 0.png" data-base62-sha1="pfn6zF6BxM6Ohx9cuyyVEWVRzyU" width="499" height="353"> </p>
<p>Like you added inside parentheses, bash {0} could work fine.  There is the official document for<a href="https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#custom-shell" rel="nofollow noopener">custom shell</a>. </p>
<p>If you want to add some custom shells to suggestions, you could share the idea in the <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="nofollow noopener">Feedback form for GitHub Actions</a>.</p>

**Comment by andreasplesch** [comment]
<p>Thanks. Let me provide feedback on the form.<br>
I was mostly struck by the “Invalid value” warning although the provided value is perfectly valid.</p>
